app/javascript/src/NewService/components/NewServiceForm.jsx

```diff
@@ -0,0 +1,145 @@
+// @flow
+
+import React, {useState, useEffect} from 'react'
+import {fetch as fetchPolyfill} from 'whatwg-fetch'
+import 'core-js/es6/promise'
+import {ServiceSourceForm, FormTemplate} from 'NewService'
+import type {SourceFormProps, FormTemplateProps} from 'NewService/types'
+
+import {createReactWrapper} from 'utilities/createReactWrapper'
+
+const BASE_PATH = '/p/admin/service_discovery'
+const NAMESPACES_URL = `${BASE_PATH}/projects.json`
```

Comment:
`NAMESPACE_URL` maybe? or why plural?

Reply:
Also, this is tightly related to that `projects` json, so might not be the right name

Reply:
In the UI the texts are `Namespace`and `Name`, that's the reason why it's called this way. But there is a discussion, maybe it will be renamed in the future to `Product`, or something similar.
But I agree, we can call this const `PROJECTS_URL` it has more sense